tests/integration/testnet-scripts/cause-doublesign.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -eux
+
+# The gaiad binary
+BIN=$1
+
+VAL_ID=$2
+
+# The chain ID
+CHAIN_ID=$3
+
+# chain's IP address prefix; 7.7.7, 7.7.8...
+# see chain config for details 
+CHAIN_PREFIX=${4:-"7.7.7"}
+
+# create directory for double signing node
+mkdir /$CHAIN_ID/validatorsybil
+cp -r /$CHAIN_ID/validator$VAL_ID/* /$CHAIN_ID/validatorsybil
+
+# clear state in validatorsybil
+echo '{"height": "0","round": 0,"step": 0,"signature":"","signbytes":""}' > /$CHAIN_ID/validatorsybil/data/priv_validator_state.json
+
+# add new key to sybil
+echo '{"priv_key":{"type":"tendermint/PrivKeyEd25519","value":"tj55by/yYwruSz4NxsOG9y9k2WrPvKLXKQdz/9jL9Uptmi647OYpcisjwf92TyA+wCUYVDOgW7D53Q+638l9/w=="}}' > /$CHAIN_ID/validatorsybil/config/node_key.json
```

Comment:
Where does this key come from?

Reply:
I copied a key I got from locally running `gaiad init`.
Any key can be used, as long as no other node in the network already uses it.

Reply:
ok, sounds good

Reply:
This is just for the network encryption, not block signing, right?

Reply:
Yes. Blocks are signed with the validator's private key